src/components/ItemDetailPage/ItemDetailPage.tsx

```diff
@@ -175,10 +176,11 @@ export default class ItemDetailPage extends React.PureComponent<Props> {
                     {item.urn}
 
                     <div className="urn-actions">
-                      <span className="link" onClick={this.handleEditURN}>
-                        {t('item.edit_urn')}
-                      </span>
-
+                      {decodeURN(item.urn).type === URNType.COLLECTIONS_THIRDPARTY ? (
```

Comment:
Maybe we can just use `isThirdParty(item.urn)` here?

Reply:
Changed it!